test/test.js

```diff
@@ -485,15 +485,15 @@ describe('serveIndex(root)', function () {
 
   describe('when using custom handler', function () {
```

Comment:
can we ensure existing tests pass as is please?  ie: make no changes to tests.

Reply:
removed the optional responseHandler and used same serveIndex.html/plain/json object notation for custom handlers.